init_workspace

```diff
@@ -51,3 +51,37 @@ if [ ${COPY_SSH} -eq 1 ]; then
 else
     echo "Not copying SSH keys from host."
 fi
+
+echo ""
+echo "---------------------------------------------------------------------------"
+echo "Checking required tools on the development host"
+echo "---------------------------------------------------------------------------"
+
+check_package() {
+    if dpkg-query -W -f='${Status}' "$1" 2>/dev/null | grep -q "install ok installed"; then
+        echo "Required package $1 is installed."
+    else
+        echo "Error: $1 is not installed."
+        echo "Please install it using 'sudo apt install $1'."
+        return 1
+    fi
+}
+
+required_packages=("python3" "python3-venv" "python-is-python3" "binfmt-support" "qemu-user-static")
```

Comment:
```suggestion
required_packages=("binfmt-support" "qemu-user-static")
```
python is not required on the host and for sure not python-is-python3..
Btw: once qemu-user is fixed (which doesn't seem likely at the moment), binfmt-support will also not be required anymore, because they changed the way qemu-user is registered with binfmt